content/tutorials/token-authentication.textile

```diff
@@ -34,13 +34,9 @@ tags:
     - Authentication
 ---
 
-Ably supports two types of authentication schemes. Basic authentication uses one of your private API keys and is the simplest scheme designed for use by your servers. Token authentication is mostly used by your device and browser clients whereby a short-lived secure token is issued to them by your servers.
+Ably supports two types of authentication schemes, basic authentication using a private API key, which should not be used on user devices, and "token authentication":/core-features/authentication#token-authentication which can be used on your server to keep your private API key secure. Your server (or serverless function) can generate short-lived tokens and pass them to users when they need to communicate with Ably. These tokens can be limited to specific capabilities on specific channels. Make sure to follow the principle of least privilege, giving clients only the capabilities they need.  
 
-Most of our "other tutorials":/tutorials demonstrate the use of the simple "basic authentication scheme":/core-features/authentication#basic-authentication. In this tutorial we are focusing on how you can use the recommended "token authentication scheme":/core-features/authentication#token-authentication for your untrusted clients.
-
-It is important to bear in mind that when using token authentication, unlike with basic authentication, the client should not be given a token at the time it is instanced. This is because the token will expire. Instead, the client should be configured with a means to obtain a token whenever it needs to authenticate with Ably. As tokens typically expire every hour, which minimizes the impact of compromise on a client device or browser, it is necessary that the client can obtain a new token to authenticate whenever the token expires or needs refreshing.
-
-Let's get started and show you a typical client-server architecture using token authentication.
+Clients should be configured with an @authCallback@ or @authUrl@, which the Ably client SDK will use to generate and renew a token whenever needed: when a token doesn't currently exist or when it has expired.
```

Comment:
Perhaps put `authUrl` first here (because this is what we're using in the tutorial) and explain that `authCallback` is mainly for non-browser clients.

Reply:
Fixed by a7c5e1f1b9873c7dffb034dec651f6e91af7c352

Though I do find `authUrl` a bit confusing/ less well documented, in terms of how the client should structure their clientOptions (`authHeaders` and `authParams`).